apps/storybook/.storybook/webpack.config.expo.js

```diff
@@ -0,0 +1,97 @@
+const webpack = require('webpack');
```

Comment:
Would the default Expo Web webpack config work here?

Reply:
no, but once I merge all of the current expo-cli PRs I'll add compatibility.